recipe/src/routes/routes.ts

```diff
@@ -69,14 +69,38 @@ export const initializeRecipeRoutes = (app: Application) => {
         }
     });
 
-    /* TODO */
+    /* gets a recipe associated to an id*/
     recipeRouter.get('/recipe/:recipe_id', async (req, res) => {
-        res.send('Test RR1');
+        try {
+            const recipe = await RecipesModel.findById(req.params.recipe_id);
+            if (recipe === null) {
```

Comment:
I'd probably also consider `res.status(404).send(...)` for better clarity on the API-user end, rather than throwing an error.

Reply:
Noted; I replaced the error and used 404 instead

Reply:
PR should be read for another review